sdks/python/apache_beam/runners/portability/stager_test.py

```diff
@@ -242,7 +242,7 @@ def test_with_main_session(self):
 
   # (BEAM-13769): Remove the decorator once cloudpickle is default pickle
   # library
-  @pytest.mark.skip
+  @pytest.mark.no_xdist
```

Comment:
Can we restore the pickler library to the default value after the test?

Reply:
Let's restore the value in addition to no_xdist.

Reply:
For context into what valentyn is ask for... do something like this:

```
try:
  staging_dir = self.make_temp_dir()
    options = PipelineOptions()
    options.view_as(SetupOptions).save_main_session = True
    # even if the save main session is on, no pickle file for main
    # session is saved when pickle_library==cloudpickle.
    options.view_as(SetupOptions).pickle_library = pickler.USE_CLOUDPICKLE
    self.update_options(options)
    self.assertEqual([],
                     self.stager.create_and_stage_job_resources(
                         options, staging_location=staging_dir)[1])
finally:
  # Revert pickle library to the default library.
  pickler.set_library(pickeler.DEFAULT)

Reply:
I think there is no need for try and final blocks. We can just set the value to default

Reply:
If the test fails, the value will not be the default after the test.  But yeah, I'm fine with either way.

Reply:
I have put this in the setUp which should achieve the same thing.

Reply:
the important point is to reset the pickler *after* the test, not before the test, so as to avoid side-effects on other tests. so you'd need to put it in tearDown